### PR TITLE
Raises Sarathi max age to 175.

### PR DESCRIPTION
**Pull request by Sadhorizon**
## About The Pull Request
Title.

## Why It's Good For The Game
Quoting [the wiki](https://shiptest.net/wiki/Sarathi), 
> Sarathi have a longer lifespan than humans, with the oldest typically reaching 175 years old.

I assume lore people forgor to update it code-wise.

## Changelog

:cl:
tweak: Sarathi max age is now 175.
/:cl:

### CODE REVIEW

**Comment by Sadhorizon** [comment]
Wuh. Why so many commits... did I click something wrong?

**Comment by Anticept** [comment]
Not an issue for this PR. You probably created a branch based on another branch that has that [b929352](https://github.com/shiptest-ss13/Shiptest/pull/2959/commits/b9293529215acac3a226c0eba18b57148b9703e4) commit, pulled on it, or edited master...

However, changed files doesnt say there's any differences so it should be a simple fix with a squash when merged.

**Comment by Sadhorizon** [comment]
> Not an issue for this PR. You probably created a branch based on another branch that has that [b929352](https://github.com/shiptest-ss13/Shiptest/pull/2959/commits/b9293529215acac3a226c0eba18b57148b9703e4) commit.

Nope, I'm just a dummy :D

File: code/modules/mob/living/carbon/human/species_types/lizardpeople.dm [hunk 1]
```diff
@@ -3,6 +3,8 @@
 	name = "\improper Sarathi"
 	id = SPECIES_SARATHI
 	default_color = "00FF00"
+	species_age_min = 17
```
Comment (meemofcourse) [inline code/modules/mob/living/carbon/human/species_types/lizardpeople.dm]:
```suggestion
	species_age_min = 18
```

Comment (Sadhorizon) [reply code/modules/mob/living/carbon/human/species_types/lizardpeople.dm]: It is actually the default for other races. This line does nothing tbqh. Ima just remove it, not sure why I added it in the first place.

**Pull request merged**